frontend/src/scenes/organization/Settings/membersLogic.tsx

```diff
@@ -21,7 +21,10 @@ export const membersLogic = kea<membersLogicType>({
         members: {
             __default: [],
             loadMembers: async () => {
-                return (await api.get('api/organizations/@current/members/')).results
+                const allMembers = (await api.get('api/organizations/@current/members/')).results
+                return allMembers.filter(
+                    (member: OrganizationMemberType) => !member.user.email.endsWith('posthogbot.user')
```

Comment:
I see two possible improvements that can be considered here:
1. We could add boolean field `is_internal` to the user model, which would make this more solid at a cost of – well – an additional field.
2. We could do this filtering in the backend, since looks like there's just no need to reveal internal (bot) users in the API.

Reply:
Does make sense.

Reply:
Went with 2. Won't add a column just yet for something that is somewhat of a workaround like bot users for api keys.